### ISSUE TITLE
Add dino to generic loading component

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#3906]
> > > Other than the one commented-out line that can probably be deleted, this is all good. Gonna miss the dino though :)
> > 
> > 
> > @schalkneethling Me too! I liked the dino.
> 
> I deliberately didn't delete the `dino.svg` icon from the `assets` folder. Hint hint.
> I think it would be pretty if that `dino.svg` could load as a backdrop to the "Loading..." message. (let's just not include it in React :)

@schalkneethling @peterbe I really like the idea of having the dino in there as a background. Let's do that if it works for you guys.

_Originally posted by @escattone in https://github.com/mdn/yari/issues/3849#issuecomment-849054083_

**Comment by tannerdolby** [comment]
@schalkneethling Does the `dino.svg` still need to be added to the generic loading component? 

I read through the thread in #3849 and was wondering if the dino.svg logo as backdrop is what currently appears when pages are loading on `mdn/yari` when running locally?

**Comment by schalkneethling** [comment]
> Does the `dino.svg` still need to be added to the generic loading component?

Yup, but with the redesign underway, perhaps we should not immediately jump on this and also get @dalexsy input. Perhaps she has something else/different in mind?

**Comment by dalexsy** [comment]
@schalkneethling As of right now, the plan is to have the redesign designed by the end of July, so I can't solidly give implementation feedback until I have a more comprehensive idea of what direction we'd be going in.

**Comment by schalkneethling** [comment]
> @schalkneethling As of right now, the plan is to have the redesign designed by the end of July, so I can't solidly give implementation feedback until I have a more comprehensive idea of what direction we'd be going in.

That is 100%, thank you @dalexsy. I more wanted you to be aware of this so it is part of the discussion as we start solidifying and implementing the redesign.
